src/main/java/SoloTech/StockFlow/stock/repository/StockRepository.java

```diff
@@ -6,5 +6,5 @@
 import java.util.Optional;
 
 public interface StockRepository extends JpaRepository<Stock, Long> {
-    Optional<Stock> findByStockId(String stockId);
+    Optional<Stock> findByStockIdAndDeletedFalse(String stockId);
```

Comment:
```suggestion
    Optional<Stock> findByStockId(String stockId);
```

엔티티에 @Where(clause = "deleted = false") 를 선언했기때문에 필요없지않을까요??

Reply:
넵 수정하겠습니다